idea-plugin/src/main/resources/localization/MetaIdeBundle_es.properties

```diff
@@ -0,0 +1,13 @@
+proofs.kotlin=Kotlin
+proofs.arrow=Λrrow
+proofs.coercion=Coercion
+proofs.type.proofs=Type Proofs
```

Comment:
Have you tried not declaring them for all languages? Feature names don't need to be localized, actually I'd say we don't want to, as explained above.